Storage/SQL/SQLiteRemoteClientsAndTabs.swift

```diff
@@ -552,13 +428,24 @@ extension SQLiteRemoteClientsAndTabs: ResettableSyncStorage {
     }
 
     public func clear() -> Success {
-        return self.doWipe { (conn, err: inout NSError?) -> Void in
-            if let error = conn.executeChange("DELETE FROM \(TableTabs) WHERE client_guid IS NOT NULL") {
+        return doWipe { conn -> Void in
+            var err: NSError? = nil
+
+            do {
+                try conn.executeChange("DELETE FROM \(TableTabs) WHERE client_guid IS NOT NULL")
```

Comment:
Remember that `ON DELETE CASCADE`? Keep this, though — it's less work for the foreign key checker.